nvm.sh

```diff
@@ -1559,7 +1564,7 @@ nvm_get_checksum() {
     SHASUMS_URL="${MIRROR}/${3}/SHASUMS.txt"
   fi
 
-  nvm_download -L -s "${SHASUMS_URL}" -o - | command awk "{ if (\"${4}.tar.${5}\" == \$2) print \$1}"
+  nvm_download -L -s "${SHASUMS_URL}" -o - | command awk "{ if (\"${4}.${5}\" == \$2) print \$1}"
```

Comment:
Why is this change needed?

Reply:
Because for Windows, you would get a .zip file, not a .tar.xxx
So I have changed it to pass the full extension, including tar.

Reply:
Argument 5 was already the extension in fact, it was the second part of the tar extension: gz or xz.
So now argument 5 would be the full extension, either tar.gz, tar.xz or zip.
I looks ok for me like that.